library/std/src/env.rs

```diff
@@ -695,21 +695,21 @@ pub struct ArgsOs {
     inner: sys::args::Args,
 }
 
-/// Returns the arguments which this program was started with (normally passed
+/// Returns the arguments that this program was started with (normally passed
 /// via the command line).
 ///
 /// The first element is traditionally the path of the executable, but it can be
 /// set to arbitrary text, and may not even exist. This means this property should
 /// not be relied upon for security purposes.
 ///
-/// On Unix systems shell usually expands unquoted arguments with glob patterns
+/// On Unix systems the shell usually expands unquoted arguments with glob patterns
```

Comment:
```suggestion
/// On Unix systems, the shell usually expands unquoted arguments with glob patterns
```

Reply:
My understanding is that commas are usually recommended after four or more words of an introductory phrase and it sounds smoother to me without the comma.